frontend/test/ui/pages/desktop/base.py

```diff
@@ -0,0 +1,17 @@
+from pypom import Page
+
+from selenium.webdriver.common.by import By
+
+
+class Base(Page):
+
+    _copter_locator = (By.CLASS_NAME, 'copter')
+
+    def __init__(self, selenium, base_url, locale='en-US', **kwargs):
+        super(Base, self).__init__(
+            selenium, base_url, locale=locale, timeout=30, **kwargs)
```

Comment:
Do we need a 30 second timeout? The default of 10 should be more than enough for most applications.